orchestrator/modules/rag/service.py

```
@@ -651,23 +651,36 @@ async def _multi_query_retrieval_with_rrf(
                     limit=limit_per_query,
                     min_similarity=min_similarity,
-                    workspace_id=workspace_id
+                    workspace_id=workspace_id,
                 )
-                
-                for rank, doc in enumerate(results):
-                    doc_id = doc.get("id", doc.get("content", "")[:100])
-                    all_results[doc_id]["ranks"].append(rank)
-                    all_results[doc_id]["doc"] = doc
-                    
             except Exception as e:
```

Ruff (0.15.20)

[warning] Do not catch blind exception: `Exception`

(BLE001)